packages/ethereum-contracts/contracts/tokens/SETH.sol

```diff
@@ -25,11 +25,16 @@ interface IWETH is IERC20 {
  * @author Superfluid
  */
 interface ISETHCustom {
+    // using native token
     function upgradeByETH() external payable;
     function upgradeByETHTo(address to) external payable;
-    function upgradeByWETH(uint wad) external;
     function downgradeToETH(uint wad) external;
-    function downgradeToWETH(uint wad) external;
+
+    // using wrapped native token
+    function getUnderlyingToken() external view returns(address tokenAddr);
+    function upgrade(uint256 amount) external;
+    function upgradeTo(address to, uint256 amount, bytes calldata data) external;
```

Comment:
I think it's possible to merge the ISETH and ISETHCustom interfaces? or is there a reason for having them separate?

Reply:
same as previous answer, the proxy conotract contains the "custom" token functions